src/containers/MyNdla/Arena/components/AdminNavLink.tsx

```diff
@@ -20,75 +19,69 @@ interface Props {
   icon: ReactNode;
 }
 
-const StyledSafelink = styled(SafeLink)`
-  color: ${colors.text.primary};
-  display: flex;
-  flex-direction: row;
-  gap: ${spacing.normal};
-  padding: ${spacing.normal};
-  padding-right: ${spacing.medium};
-  border: 1px solid ${colors.brand.light};
-  border-radius: ${misc.borderRadius};
-  box-shadow: none;
+const StyledSafelink = styled(SafeLink, {
+  base: {
+    border: "1px solid",
+    borderColor: "stroke.default",
+    borderRadius: "xsmall",
+    display: "flex",
+    flexDirection: "row",
+    gap: "medium",
+    padding: "medium",
```

Comment:
Gjør heller om til paddingInline og paddingBlock istedenfor å mikse begge 👯

Reply:
Er det noen god grunn til det annet enn at man får likere navn? Eller er det eneste vi oppnår å bytte 2 linjer med tre?

Reply:
Mer at du bruker både padding og padding-inline/block, tenker vi holder oss til kun en 👯

Reply:
Vel, det er det som er spørsmålet; er det kun for syns skyld (idk)? Når du skal sette padding på 3 sider te noe og avvike på 1, har jeg alltid satt den generelle først, og deretter lagt på avviket. 2 linjer, ferdig. Jeg syns det blir litt i overkant verbost hvis vi skal skrive dette ut til `paddingBlock: "medium", paddingInlineStart: "medium", paddingInlineEnd: "large"` bare fordi 🤷 Hvis alt det gir oss er en ekstra linje kode, og ingenting annet funksjonelt av verdi, så synes jeg ikke det er veien å gå.

Reply:
Dette går fint.